src/Field/AbstractType.php

```diff
@@ -47,9 +47,13 @@ public function render(array $attributes = [], array $viewData = [])
     {
         $this->options = array_merge($this->options, $attributes);
 
-        $value = request($this->name, $this->getOption('value'));
-
-        $this->setValue( old($this->name, $value) );
+        if (func_num_args() == 3) {
```

Comment:
I don't like the look of this code, it is incorrect as well - if you have three args, get the second as value, but the second arg is an array with view data. This is a change that will affect all form elements and I don't want that. Please describe the problem you want to fix in an issue and we can discuss how it can be fixed, but this is definitely not the right fix.